prometheus-ksonnet/mixins.libsonnet

```diff
@@ -16,6 +16,7 @@
           kubeControllerManagerSelector: 'job="kube-system/kube-controller-manager"',
           kubeApiserverSelector: 'job="kube-system/kube-apiserver"',
           podLabel: 'instance',
+          notKubeDnsCoreDnsSelector: 'job!~"kube-system/kube-dns|coredns"',
```

Comment:
nit: Is coredns running anywhere in our clusters? Depending on the configuration this might be also:
'job!~"kube-system/(kube-dns|coredns)'...

Reply:
That was not changed in #344 and simply taken from the original value from kubernetes-mixin, just changing what is strictly necessary here.